translations/hi/translation.json

```diff
@@ -0,0 +1,73 @@
+        "json": "ऐसा लगता है कि आपकी JSON फ़ाइल लोड करने में कोई समस्या थी। {{suggestion}}",
+        "large": "यदि आपकी बड़ी फ़ाइल सफलतापूर्वक नहीं लाई गई है, तो हम फ़ाइल को छोटे खंडों में विभाजित करने और उन्हें लाने की सलाह देते हैं।",
+        "strings": "ऐसा लगता है कि आपकी टेक्स्ट फ़ाइल लोड करने में कोई समस्या थी। {{suggestion}}",
+        "suggestion": "यह जाँचने का प्रयास करें कि फ़ाइल पथ ({{filePath}}) सही है या नहीं, फ़ाइल को ऑनलाइन होस्ट करना, या स्थानीय सर्वर चलाना।\n\n+ अधिक जानकारी: {{url}}",
+        "table": "ऐसा लगता है कि आपकी तालिका फ़ाइल लोड करने में कोई समस्या थी। {{suggestion}}",
+        "xml": "ऐसा लगता है कि आपकी XML फ़ाइल लोड करने में कोई समस्या थी। {{suggestion}}"
+      },
+      "friendlyParamError": {
+        "type_EMPTY_VAR": "{{location}} {{func}}() {{position}} पैरामीटर के लिए {{formatType}} की अपेक्षा कर रहा था, इसके बजाय एक खाली वैरिएबल प्राप्त हुआ। यदि जानबूझकर नहीं किया गया है, तो यह अक्सर दायरे के साथ एक समस्या है।\n\n+ अधिक जानकारी:{{url}}",
+        "type_TOO_FEW_ARGUMENTS": "{{location}} {{func}}() कम से कम {{minParams}} तर्कों की अपेक्षा थी, लेकिन केवल प्राप्त हुए {{argCount}}.",
+        "type_TOO_MANY_ARGUMENTS": "{{location}} {{func}}() {{maxParams}} से अधिक तर्कों की अपेक्षा नहीं कर रहा था, लेकिन प्राप्त हुआ {{argCount}}.",
+        "type_WRONG_TYPE": "{{location}} {{func}}() {{position}} पैरामीटर के लिए {{formatType}} की अपेक्षा कर रहा था, इसके बजाय {{argType}} प्राप्त हुआ।"
+      },
+      "globalErrors": {
+        "reference": {
+          "cannotAccess": "\n{{location}} \"{{symbol}}\" घोषणा से पहले प्रयोग किया जाता है। सुनिश्चित करें कि आपने वेरिएबल का उपयोग करने से पहले उसे घोषित कर दिया है।\n\n+ अधिक जानकारी: {{url}}",
+          "notDefined": "\n{{location}} \"{{symbol}}\" वर्तमान दायरे में परिभाषित नहीं है। यदि आपने इसे अपने कोड में परिभाषित किया है, तो आपको इसका दायरा, वर्तनी और अक्षर-आवरण जांचना चाहिए (JavaScript केस-संवेदी है)।\n\n+ अधिक जानकारी: {{url}}"
+        },
+        "stackSubseq": "└[{{location}}] \n\t {{func}}() में लाइन {{line}} से कॉल किया गया\n",
+        "stackTop": "┌[{{location}}] \n\t {{func}}() में लाइन {{line}} पर त्रुटि\n",
+        "syntax": {
+          "badReturnOrYield": "\nसिंटैक्स त्रुटि - रिटर्न किसी फ़ंक्शन के बाहर होता है। सुनिश्चित करें कि आपसे कोई कोष्ठक नहीं छूट रहा है, ताकि रिटर्न किसी फ़ंक्शन के अंदर हो।\n\n+ अधिक जानकारी: {{url}}",
+          "invalidToken": "\nसिंटैक्स त्रुटि - एक ऐसा प्रतीक मिला जिसे जावास्क्रिप्ट नहीं पहचानता या अपने स्थान पर अपेक्षित नहीं था।\n\n+ अधिक जानकारी: {{url}}",
+          "missingInitializer": "\nसिंटैक्स त्रुटि - एक कॉन्स्ट वैरिएबल घोषित किया गया है लेकिन प्रारंभ नहीं किया गया है। जावास्क्रिप्ट में, एक कॉन्स्ट के लिए इनिशियलाइज़र की आवश्यकता होती है। एक मान उसी स्टेटमेंट में निर्दिष्ट किया जाना चाहिए जिसमें वेरिएबल घोषित किया गया है। त्रुटि में पंक्ति संख्या की जाँच करें और स्थिरांक चर को एक मान निर्दिष्ट करें।\n\n+ अधिक जानकारी: {{url}}",
```

Comment:
just a suggestion here- Translation is not only about converting English to Hindi. Its about translating and conversion and translation has some difference.

Ex- for *variable, initializer, etc* many words- you just write it in hindi instead of converting it into hindi.

Please find these mistakes in your PR and let me know when those are fixed. I have seen many such mistakes in this file.

Thanks,
Aditya Rana